var/spack/repos/builtin/packages/py-entity-management/package.py

```diff
@@ -12,11 +12,8 @@ class PyEntityManagement(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/entity-management'
     git      = 'ssh://bbpcode.epfl.ch/nse/entity-management'
 
+    version('1.0.20', tag='entity-management-v1.0.20')
     version('1.0.18', tag='entity-management-v1.0.18')
-    version('1.0.16', tag='entity-management-v1.0.16')
```

Comment:
No reason to remove the old tags, no ?

Reply:
I hope they will be available in the HPC archives, but for the current set of modules they keep anyways only the last two.

Reply:
Depends on deployment frequency…

Reply:
Ah, ok, that was my other guess. Thanks @matz-e!